### DISCUSSION TITLE
CtB Fruit Ryuuta Select Screen

### DISCUSSION THREAD

**Discussion by Digitalfear117** [ppy/osu#28346] [Ideas]
I think it would be really fun to have a "Character Select Screen" for fruit ryuuta's in the game. A feature like this might not seem so important for a top player, but I think the average casual player who wants to play Catch would like it! Maybe it could be it's own panel in the Skin Editor like this, or maybe other people have better ideas for where it could be. I'm thinking you could drag images from your pc into this folder, or it could even pull ones from the skins you already have. This way you could change your ryuuta without having to make a copy of your existing skin.

![image](https://github.com/ppy/osu/assets/28713262/331a8b62-7b97-4d01-9f06-7542054dcb01)

Maybe it should just be called Fruit Catcher, I just used the legacy name here since it was shorter

**Comment by peppy** [comment]
This is already planned for every skinnable element.

**Reply by Digitalfear117** [reply]
Awesome!
